### PR TITLE
Issue #84 Responsive design for home page 

### PR DESCRIPTION
**Pull request by sandeepm96**
Some screenshots:-

Screen width - 360px
screenshot from 2017-02-28 22 57 42

Screen width - 480px
screenshot from 2017-02-28 22 57 54

Screen width - 768px
screenshot from 2017-02-28 22 58 14

Screen width - 1280px
screenshot from 2017-02-28 22 58 25

Screen width - 1366px
screenshot from 2017-02-28 22 57 25

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -33,7 +33,6 @@
     if(isFirefox === false){
       $("#set-susper-default").remove();
     }
-    $("#set-susper-default").hide();
```
Comment (nikhilrayaprolu) [inline src/index.html]: looks like you have commits of other pull

Comment (nikhilrayaprolu) [inline src/index.html]: remove these commits

Comment (sandeepm96) [inline src/index.html]: Yes I have removed those commits now. Hope its alright now. :)

**Comment by mariobehling** [comment]
Good job!

**Comment by sandeepm96** [comment]
Thank you Sir :)

**Pull request merged**